examples/module/src/gizmo/__init__.py

```diff
@@ -2,7 +2,7 @@
 This file registers the Gizmo subtype with the Viam Registry, as well as the specific MyGizmo model.
 """
 
-from viam.components.motor import *  # Need to import motor to so the component registers itself
+from viam.components.motor import *  # noqa: F403 Need to import motor to so the component registers itself
```

Comment:
nit: there's an extra `to` in the sentence: "Need to import motor TO so the component registers itself". Can remove here and below